cmd.sh

```diff
@@ -17,8 +17,13 @@ else
 fi
 
 # unpack esp-open-sdk.tar.gz in a directory that is NOT the bound mount directory (i.e. inside the Docker image)
-cp tools/esp-open-sdk.tar.* ../
-cd ..
+rm -rf /tmp/esp
+mkdir /tmp/esp
+
+export HOME=/tmp/esp
```

Comment:
Why don't you declare that before :20 and use afterwards where you currently use `/tmp/esp`?